### PR TITLE
Project mlaas4hep

### PR DESCRIPTION
**Pull request by vkuznet**

### CODE REVIEW

File: _gsocproposals/2019/proposal_MLaaS4HEP.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ project: CMS
 year: 2019
 organization: 
   - Cornell
-  - Université Clermont Auvergne
+  - CNRS/IN2P3
```
Comment (aflp91) [inline _gsocproposals/2019/proposal_MLaaS4HEP.md]: Last year Sebastien was affiliated with `LPC-Clermont` - and I think it's still right! Maybe it's simpler for you to keep this previous parameter!

Comment (vkuznet) [reply _gsocproposals/2019/proposal_MLaaS4HEP.md]: I explicitly checked with Sebastien and he told me to use CNRS/INP3, right (@sbinet )?

**Comment by vkuznet** [comment]
ok, done.

**Comment by vkuznet** [comment]
I'm closing this PR until further notice.

**Pull request closed without merging**